core-java-modules/core-java-lang-2/src/main/java/com/baeldung/methodmultiplereturnvalues/MultipleReturnValuesUsingTuples.java

```diff
@@ -0,0 +1,18 @@
+package com.baeldung.methodmultiplereturnvalues;
+
+import java.util.List;
+
+class MultipleReturnValuesUsingTuples {
+
+    static Tuple2<Integer, Coordinates> getFirstNullNearbyLocation(List<Coordinates> coordinatesList) {
+
+        int id = 55;
```

Comment:
This example is a little harder to follow as the `Coordinates` class has a `NearbyLocation` field that doesn't serve much of a purpose, and the `id` here is a meaningless value.

What if you made this example something like `getMostDistantPoint` which had to return a distance and the coordinate from the list which was furthest from some coordinate also supplied. You could implement a distance function in `Coordinates` which uses the pythagoras distance between two coordinates... It might be a good example...?